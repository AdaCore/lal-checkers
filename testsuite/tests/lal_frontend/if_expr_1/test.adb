procedure Test is
   x : Integer;
   C : Boolean;
   A : Integer;
   B: Integer;
begin
   x := if C then A else B;
end Ex1;
