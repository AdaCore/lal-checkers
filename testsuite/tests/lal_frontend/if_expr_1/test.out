Program:
  x#1 = Input()
  C#2 = Input()
  A#3 = Input()
  B#4 = Input()
  split:
    assume(C#2)
    tmp0#5 = A#3
  |:
    assume(!(C#2))
    tmp0#5 = B#4
  x#1 = tmp0#5
  end:
